test/functional/legacy-fixtures/regression/gh-711/testcafe-fixtures/index.test.js

```diff
@@ -0,0 +1,21 @@
+'@fixture GH-711';
```

Comment:
I understand. But if it actual for the new api it's better to create a test with it